schemas/oic.collection.linkslist-schema.json

```diff
@@ -7,10 +7,5 @@
 		       "$ref": "oic.collection-schema.json#/definitions/oic.collection.alllinks"
     }
   },
-  "type": "object",
-  "properties": {
-    "links":  {
-      "$ref": "#/definitions/oic.collection.alllinks"
-    }
-  }
+  "$ref": "#/definitions/oic.collection.alllinks"
```

Comment:
I don't think you can just leave a $ref floating like that.  Wrap it in an allOf block; i.e.
"allOf": [
  {"$ref": "#/definitions/oic.collection.alllinks"}
]